spec/services/stripe_payment_status_spec.rb

```diff
@@ -2,25 +2,66 @@
 
 require 'spec_helper'
 
-describe StripePaymentStatus do
+describe StripePaymentStatus, :vcr, :stripe_version do
   subject { StripePaymentStatus.new(payment) }
-  let(:payment) { build(:payment) }
+
+  let(:secret) { ENV.fetch('STRIPE_SECRET_TEST_API_KEY', nil) }
+
+  let(:credit_card) { create(:credit_card, gateway_payment_profile_id: pm_card.id) }
+
+  let(:payment_method) {
+    create(:stripe_sca_payment_method, distributor_ids: [create(:distributor_enterprise).id],
+                                       preferred_enterprise_id: create(:enterprise).id)
+  }
+
+  before { Stripe.api_key = secret }
```

Comment:
I was going to suggest that we should reset the api key back  after the end of the test. But then I realised that this key is the only key that would be used during tests, so we might as well set it once in envrionment config. 

Then I realised we already do, in `STRIPE_INSTANCE_SECRET_KEY`. Do you know why we set the test key in a special `STRIPE_SECRET_TEST_API_KEY`?